apps/bot/src/puppeteer.ts

```diff
@@ -1,10 +1,18 @@
 import puppeteer from "puppeteer";
 
 import { config } from "~/libs/modules/config/config.js";
+import { logger } from "~/libs/modules/logger/logger.js";
+import { openAI } from "~/libs/modules/open-ai/open-ai.js";
 
 const TIMEOUT_MS = 3000;
 const LAUNCH_OPTIONS = config.getLaunchOptions();
 
+try {
+	await openAI.transcribe("wrong path");
+} catch {
+	logger.info("Dummy usage of openAI module");
+}
+
```

Comment:
Add a TODO comment and inform @MrKopchick that he can use this later since he works on recording audio